pkg/resources/view_acceptance_test.go

```diff
@@ -56,27 +59,52 @@ func TestAcc_View_basic(t *testing.T) {
 	t.Cleanup(tableCleanup)
 	statement := fmt.Sprintf("SELECT id, foo FROM %s", table.ID().FullyQualifiedName())
 	otherStatement := fmt.Sprintf("SELECT foo, id FROM %s", table.ID().FullyQualifiedName())
-	comment := "Terraform test resource'"
+	comment := random.Comment()
 
-	viewModel := model.View("test", id.DatabaseName(), id.Name(), id.SchemaName(), statement)
-	viewModelWithDependency := model.View("test", id.DatabaseName(), id.Name(), id.SchemaName(), statement)
+	// generators currently don't handle lists, so use the old way
```

Comment:
nit: lists of objects, flat lists like lists of strings are handled

Reply:
Changed comment